runtime/reference/node.md

```diff
@@ -182,17 +162,18 @@ of moving components in your project. Deno also has a more secure runtime, with
 [runtime permissions](../basics/permissions.md) that allow you to control what
 your code can access.
 
-#### deno.json (optional)
+#### deno.json
 
 Deno has its own config file, `deno.json` or `deno.jsonc`, which can be used to
 configure your project. You can use it to define tasks, dependencies, path
 mappings, and other runtime configurations.
 
-#### Migrating npm scripts to deno.json (optional)
+#### Migrating npm scripts to deno.json
 
-If preferred, you can move your npm scripts over to `deno.json`, where they can
-be run using `deno task`. This allows you to manage all necessary permission
-flags and other runtime configuration in one place.
+TODO: bartlomieju this section can be removed in favor of other sections If
```

Comment:
@bartlomieju is this a question for you for for me?

Reply:
Nope, i haven't made a decision on this one yet before finishing work yesterday.